client/src/components/Search/Search.js

```diff
@@ -3,13 +3,13 @@ import './Search.css';
 import React, { useState } from 'react';
 
 import FilterList from '../FilterList/FilterList';
-import Header from '../Header/Header';
 import { InstantSearch } from 'react-instantsearch-dom';
 import ResultList from '../ResultList/ResultList';
+import SearchBar from '../SearchBar/SearchBar';
 import { searchClient } from '../../constants/algolia_config';
 
 const Search = () => {
-  const [showFilter, setShowFilter] = useState(false);
+  const [showFilter, setShowFilter] = useState(true);
```

Comment:
Comment on why we are setting this `true` would be nice.

Reply:
Imo, the variable name is self-explanatory here so it would be unnecessary to add a comment to explain it again